src/Pages/ViewList.js

```diff
@@ -1,23 +1,66 @@
-import React from 'react';
+// import { doc } from 'prettier';
+import React, { useState } from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import { db } from '../lib/firebase';
 
 const ViewList = ({ token }) => {
   // if (token === null) token = 'default';
 
+  //{someArray.map((val, i) => <input onChange={e => handleInput(e, i)}></input>)}
+  //create  <input> elements and specify their name attributes
+
   const [list, loading, error] = useCollection(db.collection(token));
 
+  //const [filter, setFilter] =useState;
+  const [inputValue, setInputValue] = useState('');
+
+  const handleChange = (e) => {
```

Comment:
This is a good example of a slight function name changing adding to the readability of this component. `handleChange`, but what change? `handleFilterChange` might be more appropriate. Same goes with the `useState` on line 15. `inputValue` isn't descriptive. You have to search around through the component to understand what this input is. `filter` and `setFilter` might be more appropriate.